Sources/Core/AST/Stmt/ConditionalCompilationStmt.swift

```diff
@@ -94,10 +94,63 @@ public struct ConditionalCompilationStmt: Stmt {
 
   }
 
+  /// Abstracts the whole condition structure and its sub-conditions.
+  public indirect enum ConditionTree: Codable {
+
+    case operand(Condition)
+
+    case and(ConditionTree, ConditionTree)
+
+    case or(ConditionTree, ConditionTree)
+
+    public enum ConditionKind: String {
+      case holdOnly
+      case skipMain
+      case skipElse
+    }
```

Comment:
```suggestion
    public enum ConditionKind: String {

      case holdOnly

      case skipMain

      case skipElse

    }
```
I'm not sure to understand what this abstraction is representing. Having docs would help.